.gitignore

```diff
@@ -17,7 +18,6 @@
 /frontend/@types/gql
 /frontend/gui_test_screenshots
 /frontend/packages/integration-tests-cypress/cypress-a11y-report.json
```

Comment:
@TheRealJon Sorry, I just noticed this, but are we able to remove this line and line 21 below as well now?